alerting_alert_rule.go

```diff
@@ -0,0 +1,84 @@
+package gapi
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"time"
+)
+
+// AlertRule represents a Grafana Alerting Alert Rule.
+type AlertRule struct {
+	Annotations  map[string]string `json:"annotations,omitempty"`
+	Conditions   string            `json:"conditions"`
+	Data         []*AlertQuery     `json:"data"`
+	ExecErrState string            `json:"execErrState"`
+	FolderUID    string            `json:"folderUid"`
+	ID           int64             `json:"id,omitempty"`
+	Labels       map[string]string `json:"labels,omitempty"`
+	NoDataState  string            `json:"noDataState"`
```

Comment:
This should also be its own string type.

Reply:
fixed